sda-auth/main.go

```diff
@@ -353,6 +355,7 @@ func main() {
 		OIDCProvider: provider,
 		htmlDir:      "./frontend/templates",
 		staticDir:    "./frontend/static",
+		pubKey:       "",
```

Comment:
```suggestion
		pubKey:       hex.EncodeToString(publicKey[:]),
```

given that this block is placed above the struct 
```
	publicKey, err := readPublicKeyFile(Config.PublicFile)
	if err != nil {
		log.Fatal("Failure to get public key: ", err)
	}
```

Reply:
resolving this and the one below since I added them